packages/server/data/items.json

```diff
@@ -5226,6 +5226,286 @@
+    "price": 100
+  },
+
+  "corn": {
+    "type": "object",
+    "name": "Corn",
+    "description": "It's corn, and it's on a cob.",
+    "price": 100
+  },
+
+  "egg": {
+    "type": "object",
+    "name": "Egg",
+    "description": "A wonderful sorta round egg, it came from a chicken, I think.",
+    "price": 100
+  },
+
+  "goldore": {
+    "type": "object",
+    "name": "Gold Ore",
+    "description": "A piece of gold ore. It's pretty heavy.",
+    "price": 100
+  },
+
+  "gooseberry": {
+    "type": "object",
+    "name": "Gooseberry",
+    "description": "A berry that grows on bushes. It's pretty tasty when cooked.",
+    "price": 100
+  },
+
+  "ironbar": {
+    "type": "object",
+    "name": "Iron Bar",
+    "description": "A bar of iron. It's pretty heavy.",
+    "price": 100
+  },
+
+  "mango": {
+    "type": "object",
+    "name": "Mango",
+    "description": "A mango, it's a fruit and it's edible",
+    "price": 100
+  },
+
+  "milkbottle": {
+    "type": "object",
+    "name": "Milk Bottle",
+    "description": "A bottle of milk. It's pretty tasty when cooked.",
+    "price": 100
+  },
+
+  "mulberry": {
+    "type": "object",
+    "name": "Mulberry",
+    "description": "A berry that grows on or in bushes.",
+    "price": 100
+  },
+
+  "oat": {
+    "type": "object",
+    "name": "Oat",
+    "description": "A grain that is commonly used in bread.",
+    "price": 100
+  },
+
+  "peach": {
+    "type": "object",
+    "name": "Peach",
+    "description": "A peach, it's a fruit and it's edible",
+    "price": 100
+  },
+
+  "pineapple": {
+    "type": "object",
+    "name": "Pineapple",
+    "description": "A pineapple, it's a fruit and it's edible",
+    "price": 100
+  },
+
+  "plum": {
+    "type": "object",
+    "name": "Plum",
+    "description": "A plum, it's a fruit and it's edible",
+    "price": 100
+  },
+
+  "quinoa": {
+    "type": "object",
+    "name": "Quinoa",
+    "description": "Hello this is the developer, I really don't know what this is even used in so I can't come up with a witty description.",
+    "price": 100
+  },
+
+  "raspberry": {
+    "type": "object",
+    "name": "Raspberry",
+    "description": "A berry that grows on or around bushes..",
+    "price": 100
+  },
+
+  "rice": {
+    "type": "object",
+    "name": "Rice",
+    "description": "You can eat this after you cook it for a bit.",
+    "price": 100
+  },
+
+  "strawberry": {
+    "type": "object",
+    "name": "Strawberry",
+    "description": "A berry that grows on bushes.",
+    "price": 100
+  },
+
+  "tinbar": {
+    "type": "object",
+    "name": "Tin Bar",
+    "description": "A bar of tin. It's pretty heavy.",
+    "price": 100
+  },
+
+  "tinore": {
+    "type": "object",
+    "name": "Tin Ore",
+    "description": "A piece of tin ore.",
+    "price": 100
+  },
+
+  "tofu": {
+    "type": "object",
+    "name": "Tofu",
+    "description": "A block of tofu. I don't know what to do with this, I'm not vegan.",
```

Comment:
tofu can be eaten as is, or cooked with other things.it absorbs the flavors of anything you cook it with, so cooking it with various broths can amplify its default bland flavor